packages/cli/src/commands/composite/merge.ts

```diff
@@ -0,0 +1,72 @@
+import { Composite, CompositeOptions } from '@glazed/devtools'
+import { Command, type CommandFlags, getArrayArg } from '../../command.js'
+import { Flags } from '@oclif/core'
+import fs from 'fs-extra'
+
+type Flags = CommandFlags & {
+  output?: string
+}
+
+export default class CompositeMerge extends Command<Flags> {
+  static strict = false
+
+  static description = 'create a Composite from other composites'
+
+  static flags = {
+    ...Command.flags,
+    'common-embeds': Flags.string({
+      char: 'e',
+      description: `'all','none' or a list of comma-separated embeds to extract from input composites into the output composite`,
+    }),
+    output: Flags.string({
+      char: 'o',
+      description: 'path to the file where the composite representation should be saved',
+    }),
+  }
+
+  async run(): Promise<void> {
+    const compositePaths = getArrayArg(this.argv, this.flags)
+
+    if (compositePaths.length === 0) {
+      this.spinner.fail('Missing list of composite file paths')
+      return
+    }
+
+    this.spinner.start('Creating the composite...')
+    try {
+      const composites = await Promise.all(
+        compositePaths.map(async (compositePath) => {
+          const encoded = await fs.readFile(compositePath, { encoding: 'utf-8' })
```

Comment:
Maybe you could use the functions from `devtools-node` here to handle this logic?